charts/pravega/templates/pravega.yaml

```diff
@@ -33,6 +33,10 @@ spec:
     {{- if .Values.externalAccess.enabled }}
     controllerServiceAccountName: {{ .Values.serviceAccount.name }}
     segmentStoreServiceAccountName: {{ .Values.serviceAccount.name }}
+    {{- if .Values.segmentStore.service.segmentStoreLoadBalancerIP }}
```

Comment:
Is that conditional needed at all? If we remove the conditional and `.Values.segmentStore.service.segmentStoreLoadBalancerIP` is null / not set, then it's just going to set it as below:
```
segmentStoreLoadBalancerIP: null`
```
Which is the same as not setting it (thus, we don't need that conditional)

Reply:
I tried doing that but I am getting this error ``` spec.pravega.segmentStoreLoadBalancerIP in body must be of type string: "null" ```